net-libs/libwebsockets/libwebsockets-3.0.0.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="C library for lightweight websocket clients and servers"
+DESCRIPTION="Libwebsockets (LWS) is a flexible, lightweight pure C library \
+for implementing modern network protocols easily with a tiny footprint, \
+using a nonblocking event loop."
+HOMEPAGE="https://libwebsockets.org/"
+COPYRIGHT="2010-2018 Andy Green"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/warmcat/libwebsockets/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="a6b611c212c52f161f70556339fdaa199b7e9b6a167c4638e086d19db75d6290"
+SOURCE_DIR="libwebsockets-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	libwebsockets$secondaryArchSuffix = $portVersion
+	lib:libwebsockets$secondaryArchSuffix = $portVersion
+	"
+
```

Comment:
```diff
 	"
-
 REQUIRES="
```
(We like to not have an empty line before `REQUIRES`. The _other_ empty lines are all OK.)
Links:
* [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)